mne/io/nihon/tests/test_nihon.py

```diff
@@ -59,3 +59,8 @@ def test_nihon_eeg():
     with pytest.warns(RuntimeWarning, match=msg):
         annot = _read_nihon_annotations(bad_fname)
         assert all(len(x) == 0 for x in annot.values())
+
+    # assert that channels with $ are 'misc'
+    picks = any([ch for ch in raw.ch_names if ch.startswith('$')])
+    ch_types = raw.get_channel_types(picks=picks)
+    assert all([ch == 'misc' for ch in ch_types])
```

Comment:
No need to turn into a list first, `all` can operate on an iterator / list comprehension directly
```suggestion
    assert all(ch == 'misc' for ch in ch_types)
```
Do any of our data files actually have a `$`-prefixed channel name?

Reply:
The Nihon file actually does have it, so let me make that actually clear, in case someone alters the file in the future.